docs/pages/reference/architecture/session-recording.mdx

```diff
@@ -129,38 +129,45 @@ Proxy Server cannot see the SSH traffic to the node. It is encrypted end-to-end
 
 ### Record at the Proxy Service
 
-In **Recording Proxy Mode**, the Proxy Service terminates (decrypts) the SSH
-connection using the certificate supplied by the client via SSH agent forwarding
-and then establishes its own SSH connection to the final destination server.
-This allows the Proxy Service to forward SSH session data to the Auth Service to
-be recorded, as shown below:
+In **Recording Proxy Mode**, the Proxy Service terminates (decrypts) SSH connections
+by presenting the client with a trusted host certificate dynamically generated by
+the Auth Service and signed by the Teleport Host CA.
 
-![recording-proxy](../../../img/recording-proxy.svg)
+The Proxy Service then establishes its own connection to the destination server authenticating in one of two ways:
+
+- For the Teleport SSH Service and legacy OpenSSH servers, the Proxy Service requires the client to provide its credentials through ssh-agent forwarding so that these credentials can be reused to connect to the OpenSSH server.
+  - Note that SSH Agent Forwarding must be allowed for any user trying to connect in this mode.
```

Comment:
```suggestion
- For the Teleport SSH Service and legacy OpenSSH servers, the Proxy Service requires the client to provide its credentials through SSH Agent forwarding so that these credentials can be reused to connect to the OpenSSH server.
  - Note that SSH Agent Forwarding must be allowed for any user trying to connect in this mode.
```